core/pv/src/main/resources/pv_ca_preferences.properties

```diff
@@ -32,3 +32,6 @@ dbe_property_supported=false
 # Mask to use for subscriptions
 # VALUE, ALARM, ARCHIVE
 monitor_mask=VALUE
+
+# Name server list
+name_servers=localhost
```

Comment:
Can we make the default empty, so there's no change for existing setups?
```
name_servers=
```
I'm not sure what otherwise happens to existing setups which would suddenly look for a name server.

Reply:
While at it, you can also make the default address list be empty, because the current setting is very specific:
```
addr_list=10.0.16.88
```

Reply:
Done!